HW1/P2/P2.txt

```diff
@@ -1 +1,9 @@
-# Your discussion here
+The histograms P2a_hist.png and P2b_hist.png show the distribution of the amount of computation carried out by each partition when running the mandelbrot calculation.  
+
+P2a_hist.png shows the effort distribution for the default partitioning strategy, while P2b_hist.png shows the effort distribution when applying partitionBy(100) during the map call that applies the mandelbrot function. The former is visibly unequally balanced, with 60 of the 100 partitions carrying out significantly less computations than the other partitions. By comparison, the latter graph is much more evenly distributed, particularly when considering that the range of the x-axis much narrower for this graph. 
```

Comment:
I'm confused...what did you do to rebalance? Running partitionBy doesn't necessarily rebalance things, right? You'd have to partition by something clever